### ISSUE TITLE
Regression in Tap stream since 1.14.4

### ISSUE THREAD

**Issue by jlstevens** [holoviz/holoviews#4957]
With 1.14.4 you can tap repeatedly to move the VLine around, but on current master it no longer works:

```python
tap = hv.streams.Tap()
def test(x, y):
    return hv.Curve([1, 2, 3]) * hv.VLine(x or 0)
dmap = hv.DynamicMap(test, streams=[tap])
dmap
```

<img src="https://user-images.githubusercontent.com/890576/119863306-06713900-bedf-11eb-8c11-ba6cca3295f0.png" width=60%></img>

**Comment by philippjfr** [comment]
This probably affects all streams so is high-priority to fix.

**Comment by douglas-raillard-arm** [comment]
Bisected to have been introduced by:
```
commit 849cd07779be75ff4e4366b3f3f13252afaa43d1
Author: Philipp Rudiger <prudiger@anaconda.com>
Date:   Sun May 23 16:00:03 2021 +0200

    Switch to Panel CommManager (#4329)
```

**Comment by jlstevens** [comment]
Thanks for running the bisection! 

Meanwhile we are working on a hotfix release (i.e. not off master branch). This bug (and related issues) will need to be fixed for 2.0.
